src/ripple/app/tests/TxQ_test.cpp

```diff
@@ -122,7 +122,7 @@ class TxQ_test : public TestSuite
     }
 
 public:
-    void run()
+    void testQueue()
```

Comment:
Not in this change set, the `return std::unique_ptr<Config const>(p.release());` on line 121 can be replaced with `return std::move(p);`. I usually point out where `return std::move` should be removed due to inhibiting RTO and just wanted to point out a case where it could be beneficially added :-)